### DISCUSSION TITLE
The width of Textbox input field for Negative Prompt Weight is so short

### DISCUSSION THREAD

**Discussion by suonnon** [lobehub/sd-webui-lobe-theme#178] [Q&A] [answered]
I'm using Lobe Theme, the text field for [Negative Prompt Weight](https://github.com/muerrilla/stable-diffusion-NPW) extension input is too short. I can't see the number :( This did not happen with default theme. Is there a way to fix this?

![image](https://github.com/canisminor1990/sd-webui-lobe-theme/assets/133345496/aca5ca2f-0a34-407c-859e-4b286d835827)

**Comment by canisminor1990** [comment]
whitch extension

**Reply by suonnon** [reply]
Sorry, It's Negative Prompt Weight
https://github.com/muerrilla/stable-diffusion-NPW

**Comment by suonnon** [comment]
Any solutions? :(
I tried to inspect the ui and change by user.css but nothing changes. I need to hard-code edit the Lobe's css style file I guess?

**Comment by canisminor1990** [comment, marked as answer]
fixed on https://github.com/canisminor1990/sd-webui-lobe-theme/commit/4fdf4a8b32a5add5d7e826024ea050ec6569d862

<img width="544" alt="image" src="https://github.com/canisminor1990/sd-webui-lobe-theme/assets/17870709/6bbe3fe0-0ab6-49bf-b791-27888fb9f34d">

**Reply by suonnon** [reply]
Yay, thank you ❤️❤️❤️
